src/pages/_app.tsx

```diff
@@ -0,0 +1,15 @@
+import type { AppProps } from "next/app";
+import { Inter } from "next/font/google";
+
+import { cn } from "@/utils/classname";
+import "@/styles/globals.css";
+
+const inter = Inter({ subsets: ["latin"], weight: ["400", "500", "600", "700"], variable: "--font-inter" });
```

Comment:
suggestion: this can be handled directly in `globals.css`

Reply:
- We're using `next/font` here to import the `inter` font and this is the way they suggest it to be used in the [docs](https://nextjs.org/docs/pages/building-your-application/optimizing/fonts). 
We can handle this in `global.css` if we were not using `next/font` and stored all the font files in public dir. 

Did I miss something?

Reply:
No, it's not a wrong approach. It's just that I personally prefer to keep this part in globals file.